### ISSUE TITLE
Example

### ISSUE THREAD

**Issue by irgusite** [zarlo/BBCodeTwigExtension#2]
Hi, I installed your extension via composer on symfony4 but it only says,: "Unknown "bbcode" filter"

Do I need to add something?

Have you an example?

Thanks :)

**Comment by zarlo** [comment]
you need to add it as a twig extension
i dont know how to do it with symfony4
https://symfony.com/doc/current/reference/dic_tags.html#enabling-custom-twig-extensions
the class is \zarlo\BBcode

**Comment by irgusite** [comment]
It works but i had to change your code a little. If you want I tidy it up and make you a pull request :)

What I did:
added these lines to services.yaml:
`zarlo\twig\BCcode:`
`    tags: [twig.extension]`
Changed the name of your class because it registered twice with ChrisKonnertz's class and symfony didn't like it. I called it BCcode 
Added use statements:
`use ChrisKonnertz\BBCode\BBCode;
use Twig\Extension\AbstractExtension;
use Twig\TwigFilter; `

The __contruct function isn't called so I called `$this->bbcode = new BBCode();` directly in bbCodeFilter.

Hope it helps :)
